src/main/java/net/fabricmc/example/ExampleMod.java

```diff
@@ -1,14 +1,18 @@
 package net.fabricmc.example;
 
 import net.fabricmc.api.ModInitializer;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class ExampleMod implements ModInitializer {
+	public static final Logger LOGGER = LogManager.getLogger();
```

Comment:
Might want to give the logger a name, for example `LogManager.getLogger("ExampleMod")`.

Reply:
By default, the getLogger will use the Caller's class name. So that would be ExampleMod anyways.

```java
// Returns a Logger with the name of the calling class.
// Returns:
//     The Logger for the calling class.
// Throws:
//     UnsupportedOperationException – if the calling class cannot be determined.
public static Logger getLogger() {
    return getLogger(StackLocatorUtil.getCallerClass(2));
}
```

Reply:
Yes, definitely mod id. It is the most useful for the end user and we want to promote best practices.

Reply:
@sfPlayer1 How would I get the mod id? Or do you mean `LogManager.getLogger("modid")`?

Reply:
Yes, but please add a comment to encourage using the actual mod id. I may add something to Loader to determine it from a class eventually, but for now it's not really supported.

Reply:
How does this sound?
```java
// This logger is used to write text to the console and the log file.
// It is considered best practice to use your mod id as the logger's name.
// That way, it's clear which mod wrote info, warnings, and errors.
public static final Logger [...]
```